src/kitchen/index.tsx

```diff
@@ -0,0 +1,14 @@
+export { Button, LinkButton } from './components/button';
+export type { ButtonProps, LinkButtonProps } from './components/button';
+
+export { Lines } from './components/lines';
+export type { LineCount, LinesProps } from './components/lines';
+
+export { Logo } from './components/logo';
+export type { LogoProps } from './components/logo';
+
+export { QuoteIcon } from './components/quote-icon';
+export type { HeadlineSize, QuoteIconProps } from './components/quote-icon';
+
+export { StarRating } from './components/star-rating';
+export type { StarRatingProps } from './components/star-rating';
```

Comment:
We decided deliberately to be explicit about what gets exposed at the top level of the `@guardian/source-kitchen-react` package. This makes the full surface area of the kitchen discoverable.